app/models/ability.rb

```diff
@@ -271,6 +271,79 @@ def initialize(user)
       can :read, Shipment
     end
 
+    #
+    # Event Organizer permissions
+    # -------------------
+    #
+    if role == 'event_organizer'
+      # Events
+      can [:read, :create], Event
```

Comment:
This [seems](https://github.com/openSUSE/travel-support-program/blob/master/app/models/ability.rb#L67) to be covered by the regular user permissions.